src/test/regress/expected/failure_create_table.out

```diff
@@ -0,0 +1,597 @@
+CREATE SCHEMA failure_create_table;
+SET search_path TO 'failure_create_table';
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- Create distributed table in 1PC and 2PC,
+-- then check if the transaction can be rollbacked.
+SET citus.shard_replication_factor TO 1;
+SET citus.shard_count to 4;
+CREATE TABLE test_table(id int, value_1 int);
+-- Kill connection before sending query to the worker 
+SELECT citus.mitmproxy('conn.kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SELECT create_distributed_table('test_table','id');
+ERROR:  connection error: localhost:57640
+DETAIL:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+SELECT count(*) FROM pg_dist_shard;
+ count 
+-------
+     0
+(1 row)
+
+\c - - - :worker_1_port
```

Comment:
To make this easier you can also run `run_command_on_workers`, that will test both workers simultaneously and saves you from having to `\connect` and reset `search_path`

Even if you decide not to use `run_command_on_workers`, maybe add a comment that you're testing that _the other_ worker wasn't given any shards. Worker 2 is the one behind the proxy, something which might not be obvious to future readers.

Reply:
Here's an example from another PR: https://github.com/citusdata/citus/pull/2280/files#diff-3ba797ec559d43345abd6c6283fd5e2dR484